content/uiux/concepts/typography/typography.md

```diff
@@ -0,0 +1,23 @@
+---
+Title: 'Typography'
+Description: 'Typography is the element of design that encompasses everything about type: how letters are arranged, the fonts designers choose, and how they are styled.'
+Subjects:
+  - 'Web Design'
+Tags:
+  - 'UI'
+  - 'UX'
+  - 'design'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Typography** is the element of design that encompasses everything about type: how letters are arranged, the fonts designers choose, and how they are styled. Making the right choices about type can help convey emotion and meaning, enhance legibility, create visual hierarchy, and shape brand identity.
+
+Users often rely on words to make sense of a product, so effective typography is a vital component of an interface. When incorporating typography in digital products, considerations include color, size, style, readability, and spacing.
+
+![The letters 'Aa' written in 15 different fonts at 100pt size, with different weight and style variations represented.](https://raw.githubusercontent.com/Codecademy/docs/main/media/typography.png)
+
+In discussing type, we’ll commonly encounter the terms “font” and “typeface.” In common usage today, the term **font** is broadly used to describe a set of characters of the same style. Historically, **typeface** referred to an entire family of fonts, such as Helvetica, Futura, or Times New Roman, while font referred to a specific size and weight of that typeface, such as “Helvetica Bold 12 pt.” Today, the terms font and typeface are often used interchangeably.
```

Comment:
```suggestion
In discussing type, the terms “font” and “typeface” are commonly encountered. In common usage today, the term **font** is broadly used to describe a set of characters of the same style. Historically, **typeface** referred to an entire family of fonts, such as Helvetica, Futura, or Times New Roman, while font referred to a specific size and weight of that typeface, such as “Helvetica Bold 12 pt.” Today, the terms font and typeface are often used interchangeably.
```

Reply:
@notalyce It's been a standard in Docs from the beginning to avoid direct address to the reader. We avoid first- and second-person pronouns.

Reply:
Yeah, Steven's correct here, on docs we use passive voice in these situations to ensure we're just speaking about the facts and there're no assumed actors. 

It's a spot where docs standards diverge from content standards.